app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala

```diff
@@ -35,6 +35,7 @@ trait AdjustedGroupInterestValidator extends BaseValidation {
     val isBetween0And100 = adjustedGroupInterestModel.groupRatio >= 0 && adjustedGroupInterestModel.groupRatio <= 100
 
     adjustedGroupInterestModel.groupRatio match {
+      case groupRatio if adjustedGroupInterestModel.qngie == 0 && groupRatio < 0 => NegativeAndQNGIEEIsZeroError(groupRatio).invalidNec
```

Comment:
This logic should actually be in the calculation validation below

Reply:
The line we want to change is this one:
`case (qngie, groupEBITA, groupRatio) if qngie / groupEBITA <= 0 && groupRatio != 100 =>`

And I think we want to change it from "<= 0" to "<0"